### ISSUE TITLE
Not working with identityseserver4 >= 3.0.0

### ISSUE THREAD

**Issue by Anat56** [Rzpeg/IdentityServer3.AccessTokenValidation#8]
I use IdentityServer3.AccessTokenValidation in my Web API. When I update my identityserver4, I get response - code 401,  "Message": "Authorization has been denied for this request." With version 2.5.3 everything works correctly.

**Comment by Rzpeg** [comment]
I don't really have enough spare time now to debug the package against v3 (We haven't migrated to v3 ourselves, given that it doesn't run on full .net framework). 

Contributions are welcomed though.

**Comment by jason-mcevoy** [comment]
I had this issue as well when migrating to IdentityServer 3.1.2 from version 2.  After the upgrade my NET framework api started rejecting access tokens with 401 responses.

What I ended up doing was setting the EmitLegacyResourceAudienceClaim to true in the identity server options.  

Once I did that, my api started working again with valid access tokens.

More info here:  https://github.com/IdentityServer/IdentityServer4/issues/3705

Hope it helps
